File "tests/runtime/invalid/try_catch_break_1.sk", line 6, characters 5-5:
'{}' is not a valid expression. Did you mean '{ void }'?
4 |     try {
5 |       break
6 |     } catch {
  |     ^
7 |     | _ -> void
